tests/execution/test-enum-empty.md

```diff
@@ -0,0 +1,18 @@
+---
+expect_validation_error: true
+---
+
+# test-enum-empty
+
+```graphql @server
+schema @server @upstream(baseURL: "http://localhost:8080") {
+  query: Query
+}
+
+enum Foo {
+}
```

Comment:
Write this config in JSON, to test for the validation error. Right now this is intecepted as invalid GraphQL before our validation logic kicks in.

Reply:
I fixed it.